src/ducks/subject-viewer.js

```diff
@@ -19,18 +19,20 @@ const MAX_SCALING = 10;
 //Initial State
 const initialState = {
   //Image transformations
+  contrast: false,
   rotation: 0,
   scaling: 1,
   translationX: 0,
   translationY: 0,
-  
+
   //Viewer settings
   viewerState: SUBJECTVIEWER_STATE.NAVIGATING,
   viewerSize: { width: 0, height: 0 },
   imageSize: { width: 0, height: 0 },
 };
 
 //Action Types
+const SET_CONTRAST = 'SET_CONTRAST';
```

Comment:
[minor] Naming suggestion: since the reducer logic is `contrast: !state.contrast`, perhaps a better name for this action is `TOGGLE_CONTRAST`